themes/bootstrap3/templates/librarycards/home.phtml

```diff
@@ -52,7 +52,11 @@
   <?php endif; ?>
 
   <div class="btn-group">
-    <a href="<?=$this->url('editLibraryCard') ?>NEW" class="btn btn-link"><i class="fa fa-edit" aria-hidden="true"></i> <?=$this->transEsc('Add a Library Card')?></a>
+    <?php if($this->shibboleth): ?>
```

Comment:
This prevents adding of a normal library card if the user has logged in using Shibboleth.